scripts/install_backend_python_libs.py

```diff
@@ -42,6 +43,24 @@ def _normalize_python_library_name(library_name):
     Returns:
         str. A normalized library name that is all lowercase.
```

Comment:
I think your docstring (above) needs to be clearer in defining normalization. You can say:

```
"""Returns a normalized version of the python library name.

Normalization means converting the library name to lowercase, and removing any "[...]" suffixes that occur. The reason we do this is ...

Here are some examples of the ambiguities that this process catches:
- (something demonstrating lowercase)
- `google-api-core[grpc]` means ..., but we don't need the [...] because ...
```

(something along those lines, perhaps)